addons/phone_validation/models/mail_thread_phone.py

```diff
@@ -43,6 +43,32 @@ class PhoneMixin(models.AbstractModel):
         help="Indicates if a blacklisted sanitized phone number is a mobile number. Helps distinguish which number is blacklisted \
             when there is both a mobile and phone field in a model.")
 
+    def search_phone_mobile_search(self, operator, value):
```

Comment:
use private method, no ?

Reply:
the method should also be accessible by the crm_lead and res_partner models which use this mixin, so it should not be private no?

Reply:
Private means methods cannot be called using scripts. But mixin may offer private methods available to all its inheriting models. So you can keep it private indeed.

Reply:
Okay, It will be kept private then.